mysql-test/suite/sys_vars/t/replicate_wild_ignore_table_basic.test

```diff
@@ -1,4 +1,8 @@
---source include/not_embedded.inc
+--source include/have_sequence.inc
+# have show_slave_status
+--source include/have_log_bin.inc
+CHANGE MASTER TO master_host='example.com';
```

Comment:
Most of the time was spent on `sys_vars.replicate_wild_ignore_table_basic` (the other files simply modified from copy-pastes), and about half were spend working around how SHOW REPLICA STATUS is empty without CHANGE MASTER TO 😵 the hard way.